gdrive

```diff
@@ -57,16 +57,19 @@ def init():
     return service
 
 def get_filename(service, file_id):
-    """Gets the file name for a file ID"""
-    filename = 'gdrive-file'
+    """Gets the file name for a file ID and describes its info"""
+    filedata = None
     try:
-        file_metadata = service.files().get(fileId=file_id, fields='name').execute()
-        filename = file_metadata['name']
-    except Exception as err:
-        print('An error happened while getting file name.')
-        print(err)
+        fields = 'id, name, size, modifiedTime, modifiedByMeTime, owners'
+        file_metadata = service.files().get(fileId=file_id, fields=fields).execute()
+        filedata = file_metadata
+    except:
+        return None
 
-    print('Filename is "%s" for ID: %s.' % (filename, file_id))
+    filename = 'gdrive-file'
+    if filedata is not None:
+        filename = filedata["name"]
+        describe_file(filedata)
```

Comment:
This would be done in the method I mentioned in the comment above.

Reply:
Treating this on the download function now, which has been changed to receive the file metadata instead of just the filename